manifests/b/BeagleSoftware/BeTL/0.1.0/BeagleSoftware.BeTL.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: BeagleSoftware.BeTL
+PackageVersion: 0.1.0
+InstallerType: inno
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
Thank you @stephengillie. I will accept it